scripts/release.ts

```diff
@@ -123,8 +119,8 @@ export default function(args: { _: string[], 'dry-run'?: boolean }, logger: Logg
     case 'patch':
       _upgrade(maybeRelease, logger);
       if (!dryRun) {
-        fs.writeFileSync(path.join(__dirname, '../versions.json'),
-                         JSON.stringify({ versions, hashes }, null, 2) + '\n');
+        fs.writeFileSync(path.join(__dirname, '../.monorepo.json'),
```

Comment:
Won't this delete all other fields in `monorepo.json` except for `packages`?

Reply:
Fixed, you were right. Thanks for catching that!